tests/infra/consortium.py

```diff
@@ -309,53 +291,57 @@ def store_current_network_encryption_key(self):
         ]
         infra.proc.ccall(*cmd).check_returncode()
 
-    def get_decrypt_and_submit_shares(self, remote_node):
-        for m in self.members:
-            with remote_node.member_client(member_id=m) as mc:
-                r = mc.rpc("getEncryptedRecoveryShare")
-
-                # For now, members rely on a copy of the original network encryption public key
-                ctx = infra.crypto.CryptoBoxCtx(
-                    os.path.join(self.common_dir, f"member{m}_kshare_priv.pem"),
-                    os.path.join(self.common_dir, f"network_enc_pubk_orig.pem"),
-                )
-
-                nonce_bytes = bytes(r.result["nonce"])
-                encrypted_share_bytes = bytes(r.result["encrypted_share"])
-                decrypted_share = ctx.decrypt(encrypted_share_bytes, nonce_bytes,)
+    def recover_with_shares(self, remote_node):
+        # TODO: Do something better here!
```

Comment:
A better comment would describe what's wrong with the current code :)

Reply:
Well spotted! Something I wanted to fix before PR-ing this but didn't. I will fix it now.